x-pack/plugins/canvas/canvas_plugin_src/renderers/advanced_filter/index.js

```diff
@@ -22,7 +22,6 @@ export const advancedFilter = () => ({
     );
 
     handlers.onDestroy(() => {
-      handlers.setFilter('');
```

Comment:
Is this the result of a bad rebase/merge? I think all the Canvas changes you have here are in master already.

Reply:
yeah, it's result of a bad rebase/merge